Common/AR/README.md

```diff
@@ -60,6 +53,15 @@ component.
   real-world coordinate
 - Eases access to ARKit, ARCore, and `LocationDataSource` delegate
   methods
+  
+### System Requirements
+
+iOS 11 64-bit is the mimumum requirement. Devices must support ARKit. For details about using ARKit, please refer to [Apple's
```

Comment:
```suggestion
iOS 11 is the minimum requirement. Devices must support ARKit. For details about using ARKit, please refer to [Apple's
```

I think we can drop the "64-bit" part. iOS 11 dropped support for 32-bit apps.

Reply:
updated